api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -125,6 +125,7 @@ public void savePatientQueue_shouldIncreaseTheNumberOfPatientQueueInList() throw
 		Assert.assertEquals(originalPatientQueueList.size() + 1, newPatientQueueList.size());
 		
 		Assert.assertEquals("20/10/2019-Unk-001", newPatientQueueList.get(0).getVisitNumber());
+		Assert.assertEquals(PatientQueue.Status.PENDING,newPatientQueueList.get(0).getStatus());
```

Comment:
@dkayiwa this tests for the change that has been made

Reply:
That is not correct. This assertion passes even without your fix.

Reply:
Okay will write another test for this.